marketplace/netlify-build/extension.json

```diff
@@ -0,0 +1,53 @@
+{
+  "id": "netlify-button",
+  "name": "Netlify Build and Preview",
+  "src": "http://localhost:1234",
+  "sidebar": true,
+  "parameters": {
+    "installation": [
+      {
+        "id": "publishKey",
+        "type": "Symbol",
+        "name": "PubNub publish key",
+        "description": "A public PubNub publish key to publish update events to Netlify",
```

Comment:
We don't "publish update events to Netlify", it's for notifying other users in the space.

I'd say "A public PubNub publish key to emit update events"